BazelExtensions/acknowledgement_merger.py

```diff
@@ -1,8 +1,7 @@
-#!/usr/bin/env python
+#!/usr/bin/python3
```

Comment:
#!/usr/bin/**env python3** ?

Reply:
I don't know anything, but this is why I landed on /usr/bin/python3: https://blogs.gnome.org/mcatanzaro/2018/02/16/on-python-shebangs/

Reply:
me neither. i just read https://github.com/pinterest/PodToBUILD/pull/200#pullrequestreview-929198622

Reply:
So I believe we should use `env` here, I dont see where in that article it says not to use `env`. I think they're mostly talking about not specifying a version of python (which this PR does specify).

On my machine:

```
/usr/bin/python3 --version
> 3.8.9

/usr/bin/env python3 --version
> 3.10.2
``

The `env` here will prefer to use any environment versions of python over the system installed one which is what most people would expect.

Reply:
You've convinced me 😅 I'll update them.